recipe/meta.yaml

```diff
@@ -34,9 +35,13 @@ test:
 
 about:
   home: https://github.com/neo4j/neo4j-python-driver
-  license: Apache-2.0
-  license_file: LICENSE.txt
   summary: Neo4j Bolt driver for Python
+  license: BSD-4-Clause AND Apache-2.0 AND OSL-2.0
+  license_file:
+    - LICENSE.APACHE2.txt
+    - LICENSE.PYTHON.txt
+    - LICENSE.txt
+    - NOTICE.txt
```

Comment:
@thebestnom, does this look right?

Reply:
Actually, I don't know how grayskull came up with BSD-4-Clause or OSL-2.0.  Should be `Python-2.0`.  I'll fix it.

Reply:
Yeah, thanks! Sorry that I didn't had time to do it myself 😅

Reply:
I have an automated tool that creates grayskull recipes for the new and old versions, takes the difference and applies that to the bot PR as a new commit.  That works well in a lot of cases but requires manual intervention sometimes (like here).  But it is still usually a good staring place.